projects/plugins/jetpack/_inc/client/state/recommendations/reducer.js

```diff
@@ -547,6 +553,13 @@ export const getProductSlugForStep = ( state, step ) => {
 	return false;
 };
 
+const shouldRecommendVaultPress = ( state, isWooCommerceRequired = false ) => {
+	const sitePlan = getSitePlan( state ).product_slug;
+	const isFree = 'is-free-plan' === getPlanClass( sitePlan );
+
+	return isFree && isWooCommerceActive( state ) === isWooCommerceRequired;
```

Comment:
This would imply that if a user has any paid Jetpack plan, not just VaultPress, that this recommendation would not be shown. I think that this is the correct behavior based on what I can find, just wanted to confirm 😄

Reply:
That is correct.